simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
This is a good point indeed. If this information is also part of CUDS then there is no need for this. However, the wrappers are supposed to be instantiated by Simulation object and this details are hidden from outside users.